jetson/nav/search/searchStateMachine.cpp

```diff
@@ -155,13 +155,14 @@ NavState SearchStateMachine::executeDriveToTarget() {
 void SearchStateMachine::publishSearchPoints() {
     // Construct vector from deque
     std::vector<Odometry> arr(mSearchPoints.begin(), mSearchPoints.end());
-    SearchPoints searchPoints{
-        .search_pattern_size  = static_cast<int32_t>(mSearchPoints.size()),
-        .points = arr
+    ProjectedPoints searchPoints{
+        .pattern_size  = static_cast<int32_t>(arr.size()),
+        .points = arr,
+        .path_type = "search"
```

Comment:
renamed from type because type is a keyword for LCM to represent the struct name